src/routes/home/Home.js

```diff
@@ -385,6 +403,15 @@ class Home extends React.Component {
       return confirmationMessage; // Webkit, Safari, Chrome etc.
     });
 
+    // TODO this was copied from ElectriCitibikes.js, consider unifying them
+    fetch('/borough-boundaries-clipped-to-shoreline.geo.json')
+      .then(response => response.json())
+      .then(boroughBoundariesFeatureCollection => {
+        this.setState({
+          boroughBoundariesFeatureCollection,
+        });
+      });
```

Comment:
this was copied from ElectriCitibikes.js, consider unifying them

Reply:
no longer needed, as of https://github.com/josephfrazier/reported-web/pull/410/commits/d8938cfb9f9aa2c1962022b38c4575ce6eb104cb